docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -93,6 +97,36 @@ To add a tag, follow these steps:
 1. Select a value from the **Select value** drop-down. This field is populated based upon the tag.
 1. Optional: Select **+** to add an additional tag.
 
+## Optional: Use Aggregate by
+
+{{% admonition type="warning" %}}
+**Aggregate by** is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.
+[Enable the `metricsSummary` feature toggle](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
+{{% /admonition %}}
+
+Using **Aggregate by**, you can calculate RED metrics (span count, erroring span count, and latency information) for `kind=server` spans recently sent to Tempo, grouped by one or more attributes.
+
+Whatever is selected in the **Aggregate by** list, you'll see every combination of present in your data.
+Each aggregate value, for example `intrinsic`:`name`, has a corresponding column in the results table.
+The RED metrics are calculated for every endpoint and user agent combinations found in your data.
+
+![Use Aggregate by to calculate RED metrics for spans and group by attributes](/static/img/docs/tempo/screenshot-traces-aggregate-by.png)
+
+The screenshot shows all of the successful HTTP `status_code` API calls against the `mystical-server` service.
+The results are shown in the same order used in **Aggregate by**.
+For example, **Aggregate by** lists `intrinsic.name` followed by `span.http.user_agent`.
+The first column in the results Table shows **name** and then **span.http.user_agent**.
+
+This capability is based on the [metrics summary API](/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/).
```

Comment:
I see we have this here but I was suggesting we bump it up since this API is off by default in Tempo and we want to let the user know ASAP that without this on, this won't work. 

Perhaps we say something like "If your Grafana Tempo datasource is connected to a Tempo database without the metrics summary API enabled, you will see an error message saying "loadprocessors not found" when you try to use the Aggregate By feature".

Reply:
Excellent point. I'll add that in.

Reply:
```suggestion
```